packages/serverpod/lib/src/server/log_manager/log_manager.dart

```diff
@@ -331,13 +331,9 @@ class SessionLogManager {
     LogSettings logSettings = _settingsForSession(session);
 
     if (session.serverpod.runMode == ServerpodRunMode.development) {
-      if (session is MethodCallSession) {
-        stdout.writeln(
-            'METHOD CALL: ${session.endpointName}.${session.methodName} duration: ${duration.inMilliseconds}ms numQueries: $_numberOfQueries authenticatedUser: $authenticatedUserId');
-      } else if (session is FutureCallSession) {
-        stdout.writeln(
-            'FUTURE CALL: ${session.futureCallName} duration: ${duration.inMilliseconds}ms numQueries: $_numberOfQueries');
-      }
+      stdout.writeln(
+        'CALL: ${session.callName} duration: ${duration.inMilliseconds}ms numQueries: $_numberOfQueries authenticatedUser: $authenticatedUserId',
+      );
```

Comment:
Question: Isn't this the same as enabling the `stdout` logger?

Reply:
Not really, these logs are more internal server logs, this PR deals with the session logs. I think we should clean up the server logging in the future too but that is out of scope for this PR.